django/core/files/storage.py

```diff
@@ -61,7 +76,7 @@ def get_valid_name(self, name):
         """
         return get_valid_filename(name)
 
-    def get_available_name(self, name):
+    def get_available_name(self, name, max_length=100):
```

Comment:
Setting it to None breaks most of the `FileStorageTests` from tests/file_storage.
What, to your opinion, would be a better solution? Changing the tests, adding `max_length` to every `self.storage.save()` call, or introducing `self.max_length` to the storage, and falling back to it when no such argument provided with `save()`?